### ISSUE TITLE
Can't enter a new credit card after first one is declined

### ISSUE THREAD

**Issue by kumar303** [mozilla/payments-ui#420]
To reproduce:
- Subscribe to a product using any [test number](https://developers.braintreepayments.com/javascript+python/reference/general/testing) to trigger a gateway error such as 'card was declined'
- See the declined error (screenshot)
- try to enter a new card number

You cannot enter a new number because the error hangs around and the Subscribe button is disabled.

![screenshot 2015-09-10 12 49 24](https://cloud.githubusercontent.com/assets/55398/9796223/a2ddf48c-57ba-11e5-889b-d92763a331d6.png)

**Comment by muffinresearch** [comment]
So one issue with this is that the spinner is started on submit. Not sure how we can turn that off. My first thought was if all the XHR returns the deferred then we can do something with that. Sadly the tokenization is all callbacks so that makes it harder in this case. Actions might be possible but seems a bit too far detached from the form to use such a generalized solution.

**Comment by kumar303** [comment]
yeah, I'm actually fixing the spinner in https://github.com/mozilla/payments-ui/issues/411 ; I have an idea for how to do it. Hopefully should be ready for r? when you read this.

**Comment by muffinresearch** [comment]
Excellent, looking at that now.
